src/components/PokeDex.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { ref } from 'vue'
+
+// Pikachu evolution list
+const pikachuevo = ref([
```

Comment:
I'll be asking before doing nonsense changes... Will be the correct syntax: "const pikachuevo = ref()" ?
I also found syntax with ref([]) but I am not sure if it serves other purposes which I will not use right now.
I will rename pikachuevo to pokemon when fixing the code since it will make sense no more.

Reply:
const pikachuevo = ref() this means that on initialization you have pikachuevo set on undefined
so same as something like ref(undefined)

ref([]) means empty array